unstructured.md

```diff
@@ -2,6 +2,7 @@
 layout: get_started
 title: Unstructured Data in Rest.li Quick Start
 permalink: /start/unstructured
+excerpt: This tutorial demonstrates how to serve unstructured binary data, such as Binary Large Object (BLOB), in a Rest.li server. We’ll define a Rest.li resource that responds with fortune reports (in PDF format) for GET requests and show you how to consume the GET response from a HTTP client.
```

Comment:
This tutorial demonstrates how to serve unstructured binary data, such as Binary Large Object (BLOB), in a Rest.li server. _It will show how_ to define a Rest.li resource that responds with fortune reports (in PDF format) for GET requests and how to consume the GET response from a HTTP client.